src/Foundation/NSUrlSessionHandler.cs

```diff
@@ -0,0 +1,716 @@
+using System;

```

Comment:
This was based on the ModernHttpClient source code so it must retain it's copyright notice and your own should be added too.